### DISCUSSION TITLE
what's wrong with my configuration?

### DISCUSSION THREAD

**Discussion by kedimuzafer** [sveltia/sveltia-cms#270] [Q&A] [answered]
I am not using netlify, so this is my configuration:
`backend:
  name: github
  repo: kedimuzafer/medrese-medrese-gerisi-vesvese
  branch: main

local_backend: true

media_folder: "static/uploads"
public_folder: "/uploads"

collections:
  - name: "posts"
    label: "Yazılar"
    folder: "user/blogs"
    create: true
    slug: "{{slug}}"
    path: '{{slug}}/index'
    extension: 'md'
    format: 'frontmatter'
    media_folder: ''
    public_folder: ''
    slug_transformer: >-
      function(slug) {
        return slug
          .toLowerCase()
          .replace(/[ğ]/g, 'g')
          .replace(/[ş]/g, 's')
          .replace(/[ı]/g, 'i')
          .replace(/[ö]/g, 'o')
          .replace(/[ü]/g, 'u')
          .replace(/[çÇ]/g, 'c')
          .replace(/[^a-z0-9]/g, '-')
          .replace(/-+/g, '-')
          .replace(/^-|-$/g, '');
      }
    fields:
      - { label: "Başlık", name: "title", widget: "string", required: true }
      - { label: "Açıklama", name: "description", widget: "string", required: true }
      - { 
          label: "Özet", 
          name: "summary", 
          widget: "string", 
          required: true, 
          default: "Bu yazı henüz bir özet içermiyor." 
        }
      - { 
          label: "Yayınlanma Tarihi", 
          name: "published", 
          widget: "datetime", 
          required: true,
          default: new Date().toISOString() 
        }
      - { 
          label: "Güncellenme Tarihi", 
          name: "updated", 
          widget: "datetime", 
          required: true,
          default: new Date().toISOString() 
        }
      - { 
          label: "Kapak Görüntüsü", 
          name: "cover", 
          widget: "image", 
          required: false,
          media_library: {
            config: {
              multiple: false,
              max_file_size: 10000000, # 10MB max
              media_folder_path: '{{slug}}',
              public_folder_path: '{{slug}}',
              default_transformations: [
                { 
                  type: 'resize', 
                  width: 1200, 
                  height: 630, 
                  fit: 'cover' 
                }
              ]
            }
          },
          hint: "Resmi yükledikten sonra, Svelte imagetools ile otomatik olarak işlenecektir. Özgün resmi yükleyin." 
        }
      - { 
          label: "Kapak Açıklaması", 
          name: "coverCaption", 
          widget: "string", 
          required: false 
        }
      - { 
          label: "Kapak Stili", 
          name: "coverStyle", 
          widget: "select", 
          options: [
            { label: "İçinde", value: "IN" },
            { label: "Üst", value: "TOP" },
            { label: "Sağ", value: "RIGHT" },
            { label: "Alt", value: "BOT" },
            { label: "Sol", value: "LEFT" },
            { label: "Yok", value: "NONE" }
          ], 
          default: "IN",
          required: true,
          hint: "Yazı listesinde kapak görüntüsünün konumu" 
        }
      - { 
          label: "Etiketler", 
          name: "tags", 
          widget: "list", 
          field: { 
            label: "Etiket", 
            name: "tag", 
            widget: "string" 
          },
          default: ["Genel"]
        }
      - { label: "İçerik", name: "body", widget: "markdown" }`

this is the error I get.
![Ekran Görüntüsü - 2024-12-09 05-34-35](https://github.com/user-attachments/assets/54ca1265-01a9-44a5-bb44-2925fb191b0d)
This is my server and I am trying to write a simple script that github send callback when something changed in the repo and backend will pull the content to the /user/blogs path. This is where my md files are.
could you please help me with this issue?

**Comment by kyoshino** [comment]
Does it work with Netlify CMS?

**Comment by kyoshino** [comment, marked as answer]
Never mind, I’ve identified the cause: The [Web Crypto API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API) used in the CMS doesn’t work because your site at `http://91.107.235.164:5173` is served over HTTP. Use a HTTPS URL to solve the issue. [Let’s Encrypt](https://letsencrypt.org/) is the easiest way to get a TLS certificate.

**Reply by kyoshino** [reply]
I have added a note to the [README](https://github.com/sveltia/sveltia-cms?tab=readme-ov-file#other-notes) about the restriction.

**Comment by kyoshino** [comment]
By the way, `slug_transformer` in your configuration doesn’t work in Sveltia CMS. I don’t think it works in Netlify/Decap CMS either. However, as mentioned in the [README](https://github.com/sveltia/sveltia-cms?tab=readme-ov-file#better-i18n-support), Sveltia CMS comes with built-in support for slug transliteration. A full list of substitutions, including some Turkish characters, can be [found here](https://github.com/sindresorhus/transliterate/blob/main/replacements.js).
